task/mgmt-gateway/src/update/sp.rs

```diff
@@ -11,19 +12,37 @@ use gateway_messages::{
     UpdateInProgressStatus, UpdateStatus,
 };
 
+cfg_if! {
+    if #[cfg(feature = "auxflash")] {
+        use drv_auxflash_api::AuxFlash;
+
+        mod auxflash;
+
+        use self::auxflash::State as AuxFlashState;
```

Comment:
It's not; I thought it was clearer than `use auxflash::...` since we are depending on other auxflash-related crates (even though those aren't named `auxflash`). I don't feel strongly about it though.

Reply:
I see `use self::` rarely enough that I did a double take, but could go either way.

Reply:
Removed as part of the stub import cleanup in 5c631443